src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        Random random = new Random();
+
+        String color = new ColorSupplier().getRandomColor();
```

Comment:
Make ColorSupplier class-level variable. See details in the common mistakes list;)